documentation/projects/proposals/search_relevancy_sandbox/20230530-implementation_plan_staging_elasticsearch_reindex_dags.md

```diff
@@ -131,7 +131,7 @@ Reindex][es_reindex_api] API.
 
 1. `source_index`: (Optional) The existing index on Elasticsearch to use as the
    basis for the new index. If not provided, the index aliased to
-   `<media_type>-full` will be used.
+   `<media_type>-filtered` will be used.
```

Comment:
Do you mind adding one note somewhere in the document that we'll need to ensure `<media_type>-filtered` can be created regularly as well, so we don't forget it when we go to tackle the implementation steps?

Reply:
Done in https://github.com/WordPress/openverse/pull/2358/commits/f2224a5cbc2eb6d54ef419b62514fa7ed54ac8a1 .